config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-periodics-upgrades.yaml.tpl

```diff
@@ -0,0 +1,57 @@
+periodics:
+{{- range $_, $upgrade := $.config.Upgrades }}
+- name: periodic-cluster-api-e2e-workload-upgrade-{{ ReplaceAll (TrimPrefix (TrimPrefix $upgrade.From "stable-") "ci/latest-") "." "-" }}-{{ ReplaceAll (TrimPrefix (TrimPrefix $upgrade.To "stable-") "ci/latest-") "." "-" }}-{{ ReplaceAll $.branch "." "-" }}
+  cluster: eks-prow-build-cluster
+  interval: 24h
+  decorate: true
+  rerun_auth_config:
+    github_team_slugs:
+    - org: kubernetes-sigs
+      slug: cluster-api-maintainers
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: {{ $.branch }}
+    path_alias: sigs.k8s.io/cluster-api
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: {{ $.config.KubekinsImage }}
+      args:
+      - runner.sh
+      - "./scripts/ci-e2e.sh"
+      env:
+      - name: ALWAYS_BUILD_KIND_IMAGES
+        value: "true"
+      - name: KUBERNETES_VERSION_UPGRADE_FROM
+        value: "{{ index (index $.versions $upgrade.From) "k8sRelease" }}"
+      - name: KUBERNETES_VERSION_UPGRADE_TO
+        value: "{{ index (index $.versions $upgrade.To) "k8sRelease" }}"
+      - name: ETCD_VERSION_UPGRADE_TO
+        value: "{{ index (index $.versions $upgrade.From) "etcd" }}"
+      - name: COREDNS_VERSION_UPGRADE_TO
+        value: "{{ index (index $.versions $upgrade.From) "coreDNS" }}"
```

Comment:
This should be upgrade.To

Probably also means that the config has to be updated